app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Kindly make sure your application asks for confirmation by email. this can be done by adding `confirmable` to your devise

Reply:
Hi @VICTOR-CHUKWU,
Thanks for the review, I made the change.